docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc

```diff
@@ -0,0 +1,156 @@
+* New support to display multiple linked guided decision tables in the Guided Decision Table Editor has been added.
+
+=== Projects Metrics dashboard
+A new dashboard is now available for every project listed in the Projects perspective. The Projects Metrics dashboard shows all of your project contribution metrics in one place. This new dashboard replaces the Contributors page, which was part of the Authoring group. For more information, see the "Projects Metrics dashboard" section of the _{PRODUCT} User Guide_.
+
+=== Team Metrics dashboard
+The Team Metrics dashboard shows all project metrics by team in a single page. This new dashboard replaces the older Contributors page, which was part of the Authoring group. For more information, see the "Team Metrics dashboard" section of the _{PRODUCT} User Guide_.
+
+=== New columns available on process instances, tasks, and jobs
+The following columns have been added to the corresponding perspectives:
+
+* Process instance list perspective: Last update and correlation key.
+* Task list: Last update, correlation key (of the associated process instance ID), process instance description (of the associated process instance).
+* Jobs perspective: Name of the associated process (if any), ID of the associated process instance (if any), Description of the associated process instance (if any).
+
+=== Advanced search filter
+You can now search for specific data by using the *Search* tab on the Jobs, Process Instances, Tasks Administration, and Execution Errors perspectives. For information about the new search feature, see the "Using the advanced search filter pages" section of the _{PRODUCT} {RELEASE} User Guide_.
+
+=== New Task List and Task Administration perspectives
+The former Task List perspective has been divided into the Tasks List and Task Administration perspectives. The Task List perspective is used by task operators to work with their assigned (or potentially assigned) tasks. The Task Administration perspective is used by administrators to manage tasks assigned to other users. This perspective is only available for users with admin or process-admin roles. It is similar to the Admin filter tab on the former Task List perspective.
+
+=== Improved validation in deployment descriptor editor
+
+The deployment descriptor editor in Business Central has been improved to validate data entered for the following:
+
+* Work item handlers
+* Event listeners
+* Marshalling strategies
+* Globals
+* Environment entries
+* Configuration
+
+Validation is performed automatically when a build is invoked or manually when a user clicks *Validate*.
+
+=== Error Handling
+The system now detects and stores execution errors such as exceptions while running a process instance or a job after the maximum number of retries has been exceeded. If an error occurs, an event is generated that contains the following data:
```

Comment:
The first sentence is ambiguous. Possible fixes:
"The system now detects and stores execution errors such as exceptions that occur while running a process instance or a job after the maximum number of retries has been exceeded."
"While running a process instance or a job, the system now detects and stores execution errors such as exceptions  after the maximum number of retries has been exceeded."
The second sentence has passive voice. Fix:
"If an error occurs, the system generates an event that contains the following data:"